aiida_zeopp/data/parameters.py

```diff
@@ -203,6 +214,13 @@ def output_links(self):
         """Return list of output link names"""
         output_links = []
         for k in self.output_keys:
-            output_links += [output_options[k][1]]
+            if not k.startswith('visVoro'):
+                output_links += [output_options[k][1]]
+            elif k == 'visVoro_allnodes':
+                output_links += [output_options['visVoro'][1]]
+            elif k == 'visVoro_accnodes':
+                output_links += [output_options['visVoro'][2]]
+            elif k == 'visVoro_nonaccnodes':
+                output_links += [output_options['visVoro'][3]]
 
```

Comment:
same as above (exception disappears)

Reply:
Here the exception could be resolved by defining the list in the output_options to have the code as:
```
@property
    def output_links(self):
        """Return list of output link names"""
        output_links = []
        for k in self.output_keys:
            for j in range(len(output_options[k][1])):
                output_links += [output_options[k][1][j]]
        return output_links
```

Reply:
I haven't tested this but how about:
```
@property
    def output_links(self):
        """Return list of output link names"""
        output_links = []
        for k in self.output_keys:
            output_links += output_options[k][1]
        return output_links
```

Reply:
It works.